quick_start.adoc

```diff
@@ -261,17 +261,20 @@ An entry under `identityProviders` has the following fields:
   * `baseDN` contains information to start the LDAP user search from. For example "cn=users,dc=example,dc=com"
   * `filter` contains optional filter to apply when searching the directory. For example "(objectClass=person)"
 - `ldap` contains the LDAP specific configurations:
-  * `url` contains the LDAP host and optional port of the LDAP server
+  * `url` contains the LDAP host and search parameters to use. The url syntax is `ldap://host:port/basedn?attribute?scope?filter`
+  * `insecure` when `false`, "ldap://" URLs are upgraded to a TLS connection using StartTLS. When `true`, "ldap://" URLS connect insecurely. Identity COnfiguration Management Operator does not support insecure=true. 
```

Comment:
Shall we focus on the `false` behavior since it's the only choice? How about
`insecure` must be `false`. ldaps:// URLs will attempt to connect using TLS, and ldap:// URLs will be upgraded to TLS. The identity configuration management operator does not support insecure=true.

Reply:
Please confirm what happens if a user omits `insecure`. Does it default to the `false` behavior?